irohad/main/irohad.cpp

```diff
@@ -82,17 +112,23 @@ int main(int argc, char *argv[]) {
                 std::chrono::milliseconds(config[mbr::LoadDelay].GetUint()),
                 keypair);
 
+  // Check if iroha daemon storage is properly set
   if (not irohad.storage) {
+    // Abort execution if not
     log->error("Failed to initialize storage");
     return EXIT_FAILURE;
   }
 
+  // Check if genesis block path was specified
   if (not FLAGS_genesis_block.empty()) {
+    // If it is so, read genesis block and store it to iroha storage
     iroha::main::BlockInserter inserter(irohad.storage);
     auto file = inserter.loadFile(FLAGS_genesis_block);
     auto block = inserter.parseBlock(file.value());
 
+    // Check if genesis block was empty
```

Comment:
`parseBlock` will verify that JSON is well formed and that it has all the fields that are required by the block structure. So it is more clear to write something like "Check that provided genesis block file was correct".